packages/remix-dev/devServer_unstable/index.ts

```diff
@@ -107,7 +107,8 @@ export let serve = async (
     console.log(`> ${cmd}`);
     let newAppServer = execa
       .command(cmd, {
-        stdio: "pipe",
+        stdio: ['ignore', 'inherit', 'inherit'],
+        shell: true,
```

Comment:
Looks like `shell: true` is [not required for this](https://github.com/remix-run/remix/issues/6504#issuecomment-1565937262)

```suggestion
```